.gitignore

```diff
@@ -102,3 +102,6 @@ ENV/
 
 # Vim's temporary files
 *.sw[ponm]
+
+# Version
+RELEASE-VERSION
```

Comment:
This should be
```
/RELEASE-VERSION
```

Otherwise this will ignore all files with that name, even in nested directories.

Reply:
:+1: